test/integration/connectionOptions.js

```diff
@@ -105,15 +105,28 @@ var externalBrowser =
   authenticator: 'EXTERNALBROWSER'
 };
 
-var externalBrowserMismatchUser =
+const externalBrowserWithShortTimeout =
```

Comment:
let's create this configuration as 

```
const externalBrowserWithShortTimeout = {
  ...externalBrowser,
  browserActionTimeout: 100,
};
```

to not copy past most of the configuration

Reply:
You wright Fixed